presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/AgentStatusUtils.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teradata.prestomanager.agent;
+
+import javax.ws.rs.core.Response;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.util.concurrent.Uninterruptibles.sleepUninterruptibly;
+import static com.teradata.prestomanager.agent.AgentFileUtils.getFileProperty;
+import static com.teradata.prestomanager.agent.CommandExecutor.executeCommand;
+import static com.teradata.prestomanager.common.RequestBuilder.newRequestBuilder;
+import static java.lang.String.format;
+import static javax.ws.rs.client.Entity.json;
+import static javax.ws.rs.core.Response.Status.OK;
```

Comment:
This is too many static imports.

`Response.Status`, `RequestBuilder`, `Entity`, and `AgentFileUtils` should be imported normally (the last mainly because `getFileProperty` isn't particularly clear when used out of context).

`format`, `executeCommand`, and `sleepUninterruptibly` are fine.